aptible/data_source_environment_test.go

```diff
@@ -0,0 +1,70 @@
+package aptible
+
+import (
+	"fmt"
+	"regexp"
+	"strconv"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/acctest"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+)
+
+func TestAccDataSourceEnvironment_validation(t *testing.T) {
+	requiredAttrs := []string{"handle"}
+	var testSteps []resource.TestStep
+
+	for _, attr := range requiredAttrs {
+		testSteps = append(testSteps, resource.TestStep{
+			PlanOnly:    true,
+			Config:      `data "aptible_environment" "test" {}`,
+			ExpectError: regexp.MustCompile(fmt.Sprintf("%q is required", attr)),
+		})
+	}
+
+	resource.Test(t, resource.TestCase{
+		PreCheck:          func() { testAccPreCheck(t) },
+		ProviderFactories: testAccProviderFactories,
+		Steps:             testSteps,
+	})
+}
+
+func TestAccDataSourceEnvironment_deploy(t *testing.T) {
+	rHandle := acctest.RandString(10)
+
+	resource.Test(t, resource.TestCase{
+		PreCheck:          func() { testAccPreCheck(t) },
+		ProviderFactories: testAccProviderFactories,
+		CheckDestroy:      testAccCheckEnvironmentDestroy,
+		Steps: []resource.TestStep{
+			{
+				Config: testAccAptibleEnvironment(rHandle),
+				Check: resource.ComposeTestCheckFunc(
+					resource.TestCheckResourceAttr("aptible_environment.test", "handle", rHandle),
+					resource.TestCheckResourceAttr("aptible_environment.test", "org_id", testOrganizationId),
+					resource.TestCheckResourceAttr("aptible_environment.test", "stack_id", strconv.Itoa(testStackId)),
+				),
+			}, {
+				ResourceName:            "aptible_environment.test",
+				ImportState:             true,
+				ImportStateVerify:       true,
+				ImportStateVerifyIgnore: []string{"org_id", "stack_id"},
```

Comment:
Why are these two attributes ignored?

Reply:
we talked about this previously on a huddle, basically if it's empty on the local state file this will get hydrated in the context of a test and fail in the test

Reply:
works without being set, but i had to leave this on the create previously if you recall in this file, which i copied: https://github.com/aptible/terraform-provider-aptible/pull/80/files#diff-02e7e7a46dad1806b98e63ea03e3610e4a10d2f40b70a8081f86ff33c1a8a7bbR93

Reply:
i'm dropping it like you suggested for this one